### PR TITLE
fixes #269 Addressed pause limit and added checks/prints for pause stuff.

### LINKED ISSUE #269: Pause/unpause and /readyteam
Pause querks

/pause then /pause will act like it's trying to pause over and over
allies /pause then axis /pause and server things axis paused instead

Pause/ready Enhancements
add playername of user that calls pause/unpause and /readyteam

### PR DESCRIPTION
**Pull request by natelo**
TODO: We still need to add pause countdown ..

### CODE REVIEW

File: src/game/g_admin.c [hunk 1]
```diff
@@ -1876,93 +1876,48 @@ void cmd_revealCamper(gentity_t *ent) {
...
-	}*/
 	if (!resume) {
-		level.paused = !PAUSE_NONE;
-		trap_SetConfigstring( CS_PAUSED, va( "%i", level.paused ));
-		AP(va("chat \"^zconsole: ^7%s has ^3Paused ^7a match!\n\"", tag));
+		if (level.paused != PAUSE_NONE) {
+			CP("print \"^jError: ^7Match is already paused^j!\n\"");
+			return;
+		}
+		G_pauseHandle(qtrue, TEAM_SPECTATOR);
```
Comment (krazykaze81) [inline src/game/g_admin.c:1897]: Why are we passing in spectator team here?

File: src/game/g_admin.c [hunk 2]
```diff
@@ -1876,93 +1876,48 @@ void cmd_revealCamper(gentity_t *ent) {
...
 			CP("print \"^jError: ^7Match is not paused^j!\n\"");
-		return;
+			return;
 		}
-
-		level.CNstart = 0; // Resets countdown if it was aborted before
-		level.paused = PAUSE_UNPAUSING;
-		AP(va("chat \"^zconsole: ^7%s has ^3Unpaused ^7a match!\n\"", tag));
-
+		G_pauseHandle(qfalse, TEAM_SPECTATOR);
```
Comment (krazykaze81) [inline src/game/g_admin.c:1904]: And here

File: src/game/g_svcmds.c [hunk 3]
```diff
@@ -928,14 +928,12 @@ L0 - Pause/Unpause
 =================
 */
 void Svcmd_Pause_f(qboolean pause) {
+	G_pauseHandle(pause, TEAM_SPECTATOR);
```
Comment (krazykaze81) [inline src/game/g_svcmds.c:931]: Don't we still need to pass in a team here?

File: src/game/g_admin.c (hunk 1 above)
Comment (natelo) [reply src/game/g_admin.c:1897]: We have now tineout limits per each playing team. E.g. If it’s set to 3 then axis/allied can only call 3 pauses per round.

Callvote or admin/ref pausing the game has no such limits so for those scenarios we are calling it as spec team so it’s not counted towards any limit.

File: src/game/g_svcmds.c (hunk 3 above)
Comment (natelo) [reply src/game/g_svcmds.c:931]: No, it’s a pub thing and not a team vote.

File: src/game/g_admin.c (hunk 2 above)
Comment (natelo) [reply src/game/g_admin.c:1904]: Explained above.

**Pull request merged**